src/Tools/dotnet-openapi/test/OpenApiTestBase.cs

```diff
@@ -19,7 +19,7 @@ public class OpenApiTestBase : IDisposable
         protected readonly ITestOutputHelper _outputHelper;
 
         // TODO: Use a more permanent URL
-        protected const string SwaggerJsonUrl = "https://raw.githubusercontent.com/glennc/clientgen/master/ConsoleClient/Server.v1.json";
+        protected const string SwaggerJsonUrl = "https://raw.githubusercontent.com/aspnet/AspNetCore/rybrande/ServiceReference/src/Tools/testassets/TestContent/swagger.json.txt";
```

Comment:
Still not okay to rely on an ephemeral URL, even if it's from our repo. What I really meant was that we shouldn't be relying on a remote file to exist on GitHub at all. I would recommend mocking the downloading instead of actually downloading remote code during the test.